packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -264,6 +310,183 @@ contract LockedGold is
     return totalNonvoting;
   }
 
+  /**
+   * Delegates CELO to delegatee
+   * @param delegatee The delegatee account.
+   * @param percentageToDelegate Percents of total CELO that will be delegated from delegatee.
+   */
+  function delegateGovernanceVotes(address delegatee, uint256 percentageToDelegate) external {
+    require(
+      percentageToDelegate > 0 && percentageToDelegate <= 100,
```

Comment:
question: often in solidity numbers have many extra zeros. so hundred is represented as `1000000000000 `or something. why straight up 100 here?

Reply:
This is a good question. I would say that mainly to limit number of delegatees that delegator can delagate to.